systemtests/tests/py3plug-fd-libcloud/testrunner-default

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
This file is missing the copyright notice alltogether.  Probably should also be 2023-2023 since its a new file.